templates/pages/admin/form/form_editor.html.twig

```diff
@@ -0,0 +1,316 @@
+            <button
+                class="btn btn-ghost-danger me-2 {{ not item.fields.is_deleted ? "d-none" : "" }}"
+                type="submit"
+                name="purge"
+                form="form-form"
+                title="{{ __("Delete permanently") }}"
+            >
+                <i class="ti ti-trash me-1"></i>
+            </button>
+        {% endif %}
+
+        {# Restore action #}
+        {% if item.canDelete() %}
+            <button
+                class="
+                    btn
+                    btn-ghost-secondary
+                    me-2
+                    {{ not item.fields.is_deleted ? "d-none" : "" }}
+                "
+                type="submit"
+                name="restore"
+                form="form-form"
+                title="{{ __("Restore") }}"
+            >
+                <i class="ti ti-trash-off me-1"></i>
+            </button>
+        {% endif %}
+
+        {% if item.fields.is_draft %}
+            {# Add action (which is an update form as we just change the is_draft field's value #}
+            <button
+                class="btn btn-primary"
+                type="submit"
+                name="update"
+                form="form-form"
+                title="{{ __("Add") }}"
+            >
+                <i class="ti ti-plus me-1"></i>
+                <span class="d-block add-label">{{ __("Add") }}</span>
+            </button>
+        {% elseif item.canUpdate() %}
+            {# Update action #}
+            <button
+                class="btn btn-primary"
+                type="submit"
+                name="update"
+                form="form-form"
+                title="{{ __("Save") }}"
+            >
+                <i class="ti ti-device-floppy me-1"></i>
+                <span class="d-block save-label">{{ __("Save") }}</span>
+            </button>
+        {% endif %}
+        </button>
+    </div>
+
+    {# Form id #}
+    <input type="hidden" name="id" value="{{ item.fields.id }}">
+
+    {# Set as non draft once saved #}
+    {% if item.fields.is_draft %}
+        <input type="hidden" name="is_draft" value="0">
+    {% endif %}
+
+    {# TODO: add bootstrap tooltips on each buttons #}
+
+    {{ call_plugin_hook(constant('Glpi\\Plugin\\Hooks::POST_ITEM_FORM'), {'item': item, 'options': params}) }}
+</form>
+
+{# Preview form modal #}
+<div
+    id="preview_form_modal"
+    class="modal modal-blur fade"
+    aria-modal="true"
+    role="dialog"
+    aria-labelledby="preview_form_modal_label"
+>
+    <div class="modal-dialog modal-lg modal-dialog-centered" role="document">
+        <div class="modal-content"></div>
+    </div>
+</div>
+
+<script>
+    /**
+     * Callback used when the form is updated
+     * Need to be a "var" to be accessible from the global scope
+     */
+    var glpi_form_ajax_update_callback = function () {
+        // Specific code enabled only if the item is a draft
+        let is_draft = {{ item.fields.is_draft }};
```

Comment:
It is better to ensure that a real boolean will be printed here. For instance, `{{ false }}` does not print anything.
```suggestion
        let is_draft = {{ item.fields.is_draft ? 'true' : 'false' }};
```